IOTATransferRestRequestTemplate.py

```diff
@@ -127,19 +124,25 @@ def create_transaction_dictionary(depth=100, request_tag=None):
 
 
 def requestData():
+    # type: () -> Optional[Text]
     request_id = register_request().bytes
     transaction = create_transaction_dictionary(request_tag=request_id)
-    iota.send_transfer(
+
+    # :todo: Populate these values and/or make API object globally-accessible.
+    api = create_iota_object(uri=uri, auth_token=auth_token, seed=seed)
+    api.send_transfer(
```

Comment:
No worries.  I want to make sure PyOTA is as easy to use as possible, so this also works as feedback for me (: